changelogs/fragments/inventory_remove_deprecated_options.yml

```diff
@@ -0,0 +1,2 @@
+breaking_changes:
+- plugins/inventory/now.py - Removed parameters ansible_host_source, named_groups and group_by (https://github.com/ansible-collections/servicenow.itsm/pull/213)
```

Comment:
```suggestion
- plugins/inventory/now.py - Removed parameters ``ansible_host_source``, ``named_groups`` and ``group_by`` (https://github.com/ansible-collections/servicenow.itsm/pull/213).
```

Reply:
Fixed in the commit https://github.com/ansible-collections/servicenow.itsm/pull/213/commits/ef4fe6537d1740f8345f48c8d0ccf94d7a1b9390.